src/Kunstmaan/AdminBundle/Entity/BaseUser.php

```diff
@@ -41,13 +41,26 @@ abstract class BaseUser extends AbstractUser
      */
     protected $googleId;
 
+    /**
+     * @var \DateTimeImmutable|null
+     * @ORM\Column(name="created_at", type="datetime_immutable", nullable=true)
```

Comment:
Mischien kunnen we hier eventueel een default value aan meegeven.
options={"default": "CURRENT_TIMESTAMP"}

op die manier hoeven we die column niet nullable te maken.

Reply:
I would rather keep this field empty for old users instead of adding a "incorrect" date. This could make debugging issues harder in the future if you want to use this audit function to trace back an issue